catalog/spatial/registry/registry-ebrim-transformer/src/main/java/org/codice/ddf/registry/converter/RegistryPackageConverter.java

```diff
@@ -437,36 +438,38 @@ private static boolean isAttributeMultiValued(String attribute, MetacardImpl met
         return multiValued;
     }
 
-    private static String getPhoneNumberString(TelephoneNumberType digits) {
-        String phoneNumber;
-        if (StringUtils.isBlank(digits.getExtension())) {
-            phoneNumber = String.format("(%s) %s", digits.getAreaCode(), digits.getNumber());
-        } else {
-            phoneNumber = String.format("(%s) %s extension %s",
-                    digits.getAreaCode(),
-                    digits.getNumber(),
-                    digits.getExtension());
-        }
-
-        return phoneNumber;
+    private static String getPhoneNumber(TelephoneNumberType digits) {
```

Comment:
Weird, we are supporting country codes but as you have pointed out we are not building it here. 
I'm fixing that up now. 

What are your thoughts on how that should be displayed?
I was thinking the following:

+1 (555) 555-5555 extension 555